### PR TITLE
Update Helm chart templates for Longhorn v1.1.0

### PR DESCRIPTION
**Pull request by PhanLe1010**
longhorn/longhorn#1906

### CODE REVIEW

File: chart/templates/clusterrole.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ rules:
   resources:
   - customresourcedefinitions
   verbs:
-  - "*"
+    - "*"
```
Comment (yasker) [inline chart/templates/clusterrole.yaml:12]: ?

File: chart/templates/clusterrole.yaml [hunk 2]
```diff
@@ -22,16 +22,28 @@ rules:
...
 - apiGroups: ["storage.k8s.io"]
   resources: ["storageclasses", "volumeattachments", "csinodes", "csidrivers"]
   verbs: ["*"]
+- apiGroups: ["snapshot.storage.k8s.io"]
+  resources: ["volumesnapshotclasses", "volumesnapshots", "volumesnapshotcontents", "volumesnapshotcontents/status"]
+  verbs: ["*"]
 - apiGroups: ["coordination.k8s.io"]
   resources: ["leases"]
-  verbs: ["*"]
+  verbs: ["get", "watch", "list", "delete", "update", "create"]
```
Comment (yasker) [inline chart/templates/clusterrole.yaml:39]: ?

File: chart/templates/clusterrole.yaml [hunk 3]
```diff
@@ -22,16 +22,28 @@ rules:
...
 - apiGroups: ["longhorn.io"]
   resources: ["volumes", "volumes/status", "engines", "engines/status", "replicas", "replicas/status", "settings",
               "engineimages", "engineimages/status", "nodes", "nodes/status", "instancemanagers", "instancemanagers/status"]
   verbs: ["*"]
+- apiGroups: ["coordination.k8s.io"]
```
Comment (yasker) [inline chart/templates/clusterrole.yaml:44]: Those are duplicated fields. We should clean them up on the manager's side.

File: chart/templates/storageclass.yaml [hunk 4]
```diff
@@ -1,18 +1,26 @@
...
-parameters:
-  numberOfReplicas: "{{ .Values.persistence.defaultClassReplicaCount }}"
-  staleReplicaTimeout: "30"
-  fromBackup: ""
-  baseImage: ""
-  {{- if .Values.recurringJobs.enable }}
-  recurringJobs: '{{ .Values.recurringJobs.jobsList | toPrettyJson | indent 2 | trim }}'
-  {{- end }}
+  name: longhorn-storageclass
+  namespace: longhorn-system
```
Comment (yasker) [inline chart/templates/storageclass.yaml:5]: Does other yaml used hardcoded namespace as well?

**Review by yasker — CHANGES_REQUESTED** [review]

File: chart/templates/clusterrole.yaml (hunk 1 above)
Comment (PhanLe1010) [reply chart/templates/clusterrole.yaml:12]: I am sorry. Copy error.

File: chart/templates/clusterrole.yaml (hunk 2 above)
Comment (PhanLe1010) [reply chart/templates/clusterrole.yaml:39]: This come from longhorn/longhorn-manager repo. Should we keep it the same as in over there?

File: chart/templates/storageclass.yaml (hunk 4 above)
Comment (PhanLe1010) [reply chart/templates/storageclass.yaml]: Thank you! Fixed

File: chart/templates/clusterrole.yaml (hunk 3 above)
Comment (PhanLe1010) [reply chart/templates/clusterrole.yaml:41]: Thank you. Fixed

**Review by yasker — APPROVED** [review]

**Comment by yasker** [comment]
LGTM. Waiting for the other PRs to be ready.

**Comment by PhanLe1010** [comment]
Removed the storageclass configmap from `manager` daemonset's volumes  because we will read the configmap directly from Kubernetes API.

**Pull request merged**